### DISCUSSION TITLE
v18.0.6

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1004] [General]
## Releases Notes for 18.0.6
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v18.0.6/draw.io-18.0.6-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v18.0.6/draw.io-18.0.6-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v18.0.6/draw.io-universal-18.0.6.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v18.0.6/drawio-amd64-18.0.6.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v18.0.6/drawio-x86_64-18.0.6.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v18.0.6/drawio-x86_64-18.0.6.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Security issues addressed:
	https://huntr.dev/bounties/b242e806-fc8c-41c0-aad7-e0c9c37ecdee/
	https://huntr.dev/bounties/fded4835-bd49-4533-8311-1d71e0ed7c00/
	
- Updates to [draw.io core 18.0.6](https://github.com/jgraph/drawio/blob/v18.0.6/ChangeLog). All changes from 18.0.5 to 18.0.6, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v18.0.6'>v18.0.6</a>.</em>

**Comment by wdrobn** [comment]
Hello,

I'm new to draw.io, and wanted to install the windows version. However, whenever I try running the windows installer, no matter the version, I get an error message saying "draw.io cannot be closed." (See attached image). I never have draw.io open, so I'm confused as to why this message is popping up. Hitting "retry" will just result in the same message popping up. And hitting "cancel" will just end the installation process. 

Any thoughts on what the problem may be? 

Thanks!

![drawio](https://user-images.githubusercontent.com/106308922/170460418-98d11a0e-2d2d-4b6b-b472-a74ed80bee26.png)

**Reply by davidjgraph** [reply]
Are you attempting to install on a system where you don't have admin permissions?

**Reply by wdrobn** [reply]
Yes.

**Reply by davidjgraph** [reply]
Use the no installer version if you don't have admin permissions.

**Reply by wdrobn** [reply]
That does work. But what if I want to install?

**Reply by davidjgraph** [reply]
Then you need admin permissions...
